packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -268,6 +269,13 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to being adapted to the current [TargetPlatform].
   final bool centerTitle;
 
+  /// The margin between [title] and [leading] content. This margin is applied
+  /// even if there is no leading content. If you want [title] to take all the
+  /// space available, set this value to 0.0.
+  ///
+  /// Defaults to the specification in [NavigationToolbar].
```

Comment:
(here and below) i don't know what "Defaults to the specification in [NavigationToolbar]" means.

Reply:
Changed to "Defaults to material specification." We could expose the 16dp const here I suppose since that is a material const. Let me know what you prefer.

Reply:
The more detailed the docs, the better. :-) I would totally mention the 16px explicitly.

Reply:
Done. Now that the const is public, I put a link to it.